tcms/testruns/views.py

```diff
@@ -861,8 +851,6 @@ def get_context_data(self, **kwargs):
             'test_case_run_bugs': bug_ids,
             'mode_stats': mode_stats,
             'summary_stats': summary_stats,
-            'jira_url': jira_url,
-            'bugzilla_url': bugzilla_url
```

Comment:
Well again this is broken by design. I use GitHub and tons of others bug trackers and the result of the above reporting URL is:

`https://bugzilla.redhat.com/buglist.cgi?.....` listing my GitHub issue numbers as parameters. Hardly useful to anyone outside Red Hat, won't you agree ?